modules/meta.analysis/R/meta.analysis.R

```diff
@@ -9,7 +9,7 @@
 
```

Comment:
@infotroph, I think we should remove these outdated license blocks before a release. Should I remove all of them for the `meta.analysis` package and make a commit or doing it as a separate PR would be better ?

Reply:
@meetagrawal09 It should be a separate PR that will merge into branch `release/v1.8.0` instead of into `develop`. We'll need to double-check that all code owners have signed off on the license change before we merge, but do go ahead and open the PR please.